### DISCUSSION TITLE
Tileset Coverage Poll Results

### DISCUSSION THREAD

**Discussion by I-am-Erk** [I-am-Erk/CDDA-Tilesets#858] [General]
The other day I generated a poll to assess what the community considers important for "complete" tileset coverage, as I think it differs a bit from what perfectionist artists want. https://forms.gle/pLLbhBQvyY67Du1EA

This is a summary of results after a few days and >300 results. Thanks to all that contributed their thoughts!

# Part 1: Basic coverage and mods
![image](https://user-images.githubusercontent.com/45136638/131257241-9e2b686a-dfff-4cfe-aced-e87a4b40e7f3.png)
![image](https://user-images.githubusercontent.com/45136638/131257247-c5c24ffc-a2c1-4bb9-ad27-184083c3e4c2.png)
![image](https://user-images.githubusercontent.com/45136638/131257250-6a562dae-9791-48f1-91e4-60e491ed7ca4.png)

To no one's real surprise, most people consider coverage of vanilla tiles to be quite important. I was surprised to see how important packaged mods are to a lot of users (for a while they were beating vanilla tiles for importance), and also surprised how many people consider ASCII tiles to be a dealbreaker.

# Part 2: Quantity and quality
![image](https://user-images.githubusercontent.com/45136638/131257313-732a9797-36a8-4713-8e80-393f45962ed5.png)
![image](https://user-images.githubusercontent.com/45136638/131257319-f30a3379-cc0b-4259-8992-db4d68baa227.png)

This confirms a suspicion of mine, that most users care less about the quality and uniqueness of a tile than that it is present at all, and it represents a significant difference of opinion between artists (at least this particular artist) and users. Personally, I feel like a copied or mildly edited tile is at best a stopgap filler, and barely better than an ascii fallback. I knew that wasn't a widely shared opinion, but the data suggests it's even less important than I thought.

# Part 3: Where to focus
![image](https://user-images.githubusercontent.com/45136638/131257409-549c08f3-5ef6-4ea5-a890-5b51b4bd0cf6.png)

Far and away, people want monsters. That fits with my own expectations (and it's a bit unfortunate for newer tilesets, as monsters are one of the harder things to add). I'm surprised "items on the ground" is as important as this to some people, since unless the item is on its own, you're still going to need to examine the stack, so tiles aren't very useful at conveying info, but the people have spoken.

It's a good thing I included a write-in, that sure led to some helpful answers.

# Part 4: Fill-in comments
There wasn't much of a theme to the write-in comments at the end, except that many people emphasized that a coherent theme was important to them. I'm on board with that, it's a big part of why I started UltiCa. The rest of the replies mostly reiterated the above points.

# Discussion as regards UltiCa specifically
These data are tileset agnostic, so other tileset curators can use them as they please.

My take-home here for UltiCa in specific is that we should do a final crunch to get the last monster sprites completed, even if it means lower quality than I like. Recoloured and slightly edited sprites can be used as fillers until a better option is available, as long as they are visually distinct, and the vast majority of players consider that much more important than making sure that each monster is nicely made on its own. This should mean we have full coverage very quickly, as we already have something in the vicinity of 80% of monsters covered. Packaged mods are more important than I thought, so I will encourage development of sprites (especially monsters) for Magiclysm, Aftershock, Dinomod, etc; and I am relieved to see that my opinions on third party mods are mostly reflected in the community. Those will remain on the backburner, which is good because that's a hard-to-assess pile of content that would draw focus off our efforts a great deal.

To the seven or eight people that won't use a tileset if they see an ascii character ever, regardless of if it's vanilla, packaged mod, or random mod you downloaded from a cached version of a geocities page: keep dreamin', folks, but I love your optimism.

**Comment by acepleiades** [comment]
This is very useful to know, thank you for doing this. I recently added a lot of monster sprites to the tileset, I will continue doing that after working on some other projects
